ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -190,6 +209,38 @@ def yield_tag(self, _: str) -> Iterable[OMetaTagAndClassification]:
             logger.debug(traceback.format_exc())
             logger.warning(f"Skipping Policy Tag: {exc}")
 
+    def yield_database_schema(
+        self, schema_name: str
+    ) -> Iterable[CreateDatabaseSchemaRequest]:
+        """
+        From topology.
+        Prepare a database schema request and pass it to the sink
+        """
+
+        database_schema_request_obj = CreateDatabaseSchemaRequest(
+            name=schema_name,
+            database=self.context.database.fullyQualifiedName,
+            description=self.get_schema_description(schema_name),
+        )
+
+        dataset_obj = self.client.get_dataset(schema_name)
+        if dataset_obj.labels:
+            for label_classification, label_tag_name in dataset_obj.labels.items():
+                database_schema_request_obj.tags = [
+                    TagLabel(
+                        tagFQN=fqn.build(
+                            self.metadata,
+                            entity_type=Tag,
+                            classification_name=label_classification,
+                            tag_name=label_tag_name,
+                        ),
+                        labelType="Automated",
```

Comment:
@ayush-shah , could you please import this value from `tagLabel.py` or we can keep define this value as constant

Reply:
got it, let me pull that up from tagLabel